quantum/quantum.c

```diff
@@ -209,7 +217,7 @@ bool pre_process_record_quantum(keyrecord_t *record) {
             true)) {
         return false;
     }
-    return true; // continue processing
+    return pre_process_record_quantum_kb(record);
```

Comment:
might be worth: 

```c
bool pre_process_record_quantum(keyrecord_t *record) {
    uint16_t keycode = get_record_keycode(record, true);
    if (!(
#ifdef COMBO_ENABLE
            process_combo(keycode, record) &&
#endif
            true)) {
        return false;
    }
    return pre_process_record_quantum_kb(keycode, record);
}
```
(since can't suggest for the block)

Reply:
```c
bool pre_process_record_quantum(keyrecord_t *record) {
    uint16_t keycode = get_record_keycode(record, true);
#ifdef COMBO_ENABLE
     if (process_combo(keycode, record)) return false;
#endif
    return pre_process_record_quantum_kb(keycode, record);
}
```
would be a more readable suggestion